tests/console/zypper_ref.pm

```diff
@@ -18,6 +19,12 @@ use utils 'zypper_call';
 
 sub run {
     select_console 'root-console';
+
+    # see FATE#325541
+    zypper_call 'mr -e -l'
```

Comment:
That's not the third place we got it =(. Would you be the one who can put single implementation?
`tests/autoyast/repos.pm`
`tests/sles4sap/patterns.pm` 
are the other 2 places...

Reply:
@rwx788 indeed, thought about creating a follow-up ticket to do the change.

Reply:
One way or another, feel free to merge this PR.

Reply:
I guess I'll merge for now, pick the followup later in the sprint

Reply:
Sure, let's refactor later on